### PR TITLE
Reuse of token in TOTP is possible

### PR DESCRIPTION
**Pull request by mabartos**
Documentation PR for this PR [#13867](https://github.com/keycloak/keycloak/pull/13867)

### CODE REVIEW

File: server_admin/topics/authentication/otp-policies.adoc
Comment (andymunro) [inline server_admin/topics/authentication/otp-policies.adoc]: we always spell out admin (administrator) unless it is an adjective, such as "Admin" console

Comment (mabartos) [reply server_admin/topics/authentication/otp-policies.adoc]: Thanks

**Review by mposolda — APPROVED** [review]
@mabartos Thanks, I am approving. IMO this is ok to merge once the corresponding codebase PR is merged: https://github.com/keycloak/keycloak/pull/13867

**Pull request merged**
